docs/pymodel.py

```diff
@@ -0,0 +1,293 @@
+import concurrent.futures
```

Comment:
pyotherside currently has no infrastructure to ship pure Python sublibraries to be available to library users.

`import pyotherside` provides a C library. Maybe we could make this code available under `import pyotherside.model`? This would need some work to implement, I think. If this sounds OK to you, shall I look into what it would take to make this available under such an interface?